app/namespace.go

```diff
@@ -1490,6 +1552,42 @@ func NewNamespaceCommand(getNamespaceClientFn GetNamespaceClientFn) (CommandOut,
 				return c.failoverNamespace(ctx)
 			},
 		},
+		{
+			Name:    "update",
+			Usage:   "Update Temporal namespace attributes",
+			Aliases: []string{"u"},
+			Flags: []cli.Flag{
+				RequestIDFlag,
+				&cli.StringFlag{
+					Name:     NamespaceFlagName,
+					Usage:    "The namespace hosted on temporal cloud",
+					Aliases:  []string{"n"},
+					Required: true,
+				},
+				&cli.BoolFlag{
```

Comment:
Just to confirm what command does the user to re enable managed fail over?

Reply:
`tcld namespace update -n <namespace with account id> --disable-auto-failover=false`

Reply:
Ack makes sense.